lib/kb_blast/kb_blastImpl.py

```diff
@@ -19,14 +19,15 @@
 from Bio.SeqRecord import SeqRecord
 from Bio.Alphabet import generic_protein
 #from biokbase.workspace.client import Workspace as workspaceService
-from Workspace.WorkspaceClient import Workspace as workspaceService
+#from Workspace.WorkspaceClient import Workspace as workspaceService
```

Comment:
You can delete these unused imports

Reply:
sure.  I tend to accumulate cruft.  comes from being old and learning how to code before version control systems took over.

Reply:
removed comment lines